src/style.css

```diff
@@ -1 +1,64 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.page {
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.search-bar::placeholder {
+  color: #3d4e61;
+}
+
+.search-bar__input {
+  background: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+}
+
+.search_bar__input--big {
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  background-size: 19px;
+  background-position: left 26px center;
+}
+
+.search-bar__input--small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  background-size: 11px;
+  background-position: 13px center;
+}
+
+.search_bar__input--big:hover,
```

Comment:
There is an inconsistency in class naming. The class `.search_bar__input--big:hover` uses an underscore instead of a hyphen. It should be `.search-bar__input--big:hover` to maintain consistency with the BEM naming convention used elsewhere.